example/zeto/README.md

```diff
@@ -2,35 +2,80 @@
 
 This example demonstrates a privacy-preserving fungible tokens using [Zeto](https://github.com/hyperledger-labs/zeto) on Paladin.
 
-See the [tutorial](https://lf-decentralized-trust-labs.github.io/paladin/head/tutorials/zeto/) for a detailed explanation.
+See the [tutorial](https://lf-decentralized-trust-labs.github.io/paladin/head/tutorials/zkp-cbdc/) for a detailed explanation.
 
 ## Pre-requisites
 
-Requires a local 3-node Paladin cluster running on `localhost:31548`, `localhost:31648`, and `localhost:31748`.
+### Local Paladin network
 
-## Run standalone
+This sample requires a local 3-node Paladin network running on `localhost:31548`, `localhost:31648`, and `localhost:31748`. You can get one setup by referring to the [Getting Started](https://lf-decentralized-trust-labs.github.io/paladin/head/getting-started/installation/) guide.
 
-Build [TypeScript SDK](../../sdk/typescript):
+This should provision a local kubernetes (based on Kind) cluster and provision a Besu and Paladin network including these pods:
 
-```shell
-cd ../../sdk/typescript
-npm install
-npm run abi
-npm run build
+```console
+$ kubectl get po
+NAME                               READY   STATUS    RESTARTS   AGE
+besu-node1-0                       1/1     Running   0          19m
+besu-node2-0                       1/1     Running   0          19m
+besu-node3-0                       1/1     Running   0          19m
+paladin-node1-0                    2/2     Running   0          17m
+paladin-node2-0                    2/2     Running   0          17m
+paladin-node3-0                    2/2     Running   0          17m
+paladin-operator-bc788db4f-mzbs7   1/1     Running   0          19m
 ```
 
-Run example:
+### ABI resources for Zeto
 
-```shell
-npm install
-npm run start
+The sample uses the following ABIs to accomplish the end to end flow:
+
+- IZetoFungible.json: private transaction ABI for the Zeto domain, to conduct private transactions like `mint`, `transfer`, `lock`, etc.
+- Zeto_Anon.json: public transaction ABI for the Zeto token implementation, to conduct public transactions "delegateLock`
+- SampleERC20.json: public transaction ABI for a sample ERC20 token, to conduct public transactions to mint ERC20 tokens
+
+These dependency resources can be obtained by running the following builds:
+
+```console
+$ ./gradlew :sdk:typescript:build
+$ ./gradlew :example:zeto:build
```

Comment:
You don't need to run both (the example build depends on the SDK build so it will trigger everything).

This is less of a pivot than the first pass, but is still asserting that you need to get the Gradle build working (even though you don't need to build the entire tree).

Reply:
this has now been updated to eliminate the need to run gradle at all. but instead only `npm run` from inside `sdk/typescript/scripts` and `example/zeto/scripts`, similar to the flow for the Noto example.